app/controllers/provider_interface/provider_users_controller.rb

```diff
@@ -0,0 +1,47 @@
+module ProviderInterface
+  class ProviderUsersController < ProviderInterfaceController
+    before_action :requires_provider_add_provider_users_feature_flag, only: %i[new create]
+
+    def index
+      @provider_users = ProviderUser.visible_to(current_provider_user)
+    end
+
+    def new
+      unless current_provider_user.can_manage_users?
+        flash[:warning] = 'You need specific permissions to manage other providers.'
+        return redirect_to provider_interface_provider_users_path
+      end
+
+      @form = ProviderUserForm.new(current_provider_user: current_provider_user)
+    end
+
+    def create
+      @form = ProviderUserForm.new(
+        provider_user_params.merge(current_provider_user: current_provider_user),
+      )
+      provider_user = @form.build
+      render :new && return unless provider_user
```

Comment:
This renders `204 No content` which effectively hides all the errors. Propose
```suggestion
      render :new and return unless provider_user
```

Reply:
Fixed in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1823/commits/8d3f0bbb122f763fcc290ce4e83f78b630e3bae0 I did wonder where the correct place to test for these error cases is. It seems like we may bloat system specs testing for validations but testing errors in controller flow doesn't seem to have a common place in our tests...